rx-java/src/main/java/examples/RxifiedExamples.java

```diff
@@ -66,19 +68,25 @@ public void scheduler(Vertx vertx) {
 
   public void schedulerHook(Vertx vertx) {
     RxJavaSchedulersHook hook = io.vertx.rxjava.core.RxHelper.schedulerHook(vertx);
-      RxJavaHooks.setOnIOScheduler(f -> hook.getIOScheduler());
-      RxJavaHooks.setOnNewThreadScheduler(f -> hook.getNewThreadScheduler());
-      RxJavaHooks.setOnComputationScheduler(f -> hook.getComputationScheduler());
+    RxJavaHooks.setOnIOScheduler(f -> hook.getIOScheduler());
+    RxJavaHooks.setOnNewThreadScheduler(f -> hook.getNewThreadScheduler());
+    RxJavaHooks.setOnComputationScheduler(f -> hook.getComputationScheduler());
+  }
+
+  public void workerExecutorScheduler(Vertx vertx) {
```

Comment:
it should really pass a WorkerExecutor and not try to do something on behalf of the user, the user pass the executor and has full control of its creation